packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -277,10 +246,12 @@ struct Field {
   1: required Expression expression,
   // TODO only two kinds of distinguishable annotations are before `aggregate:` and before `name is value`
   // between `name` and `is`, or between `is` and `value` are converted to before `name`.
+  // TODO consider having an annotation tree with inheritance
   2: optional list<Annotation> annotations,
 }
 
 struct OrderBy {
+  // TODO consider making this a specialized reference like an "OrderByReference"
   1: required Reference field_reference,
   2: optional OrderByDirection direction,
 }
```

Comment:
Re: https://github.com/malloydata/malloy/pull/2327/files#diff-9d0111becc433e2a89f480d46840047e845f6816c95d1b0ecd573ca173e5827eR263-R266

Currently, a couple trivial wrapper structs exist to wrap around a `union` so that it can use used in a different `union`. This is because the current TS types flatten unions. Is this okay, or should we find a way to make the TS types work with union branches being typed as unions somehow?